packages/clients/package.json

```diff
@@ -53,18 +64,28 @@
     "superstruct": "^1.0.3"
   },
   "devDependencies": {
+    "@aptos-labs/ts-sdk": ">=1.13.1",
+    "@mysten/sui.js": ">=0.46.1",
     "@tsconfig/recommended": "^1.0.3",
     "@types/uuid": "^9.0.2",
-    "@typescript-eslint/eslint-plugin": "^6.7.4",
-    "@typescript-eslint/parser": "^6.7.4",
-    "eslint": "^8.51.0",
+    "eslint": "^8.57.0",
     "jest": "^29.5.0",
     "ts-jest": "^29.1.0",
     "ts-node": "^10.9.1",
     "typescript": "^5.2.2",
+    "typescript-eslint": "^7.7.1",
     "uuid": "^9.0.0"
   },
   "peerDependencies": {
+    "@aptos-labs/ts-sdk": ">=1.13.1",
```

Comment:
The Aptos SDK is now on 1.15.0 with a couple breaking changes (not sure if they affect any tests).

The Sui SDK is now on 0.54.1

Reply:
Here we only need to specify the minimum supported versions. I manually tested with the latest Aptos SDK and it should be fine. We have a regular [CI job](https://github.com/shinamicorp/shinami-typescript-sdk/actions/workflows/dep_compat.yaml) to help detect true breaking changes.